### PR TITLE
[ODS-4778] Deploy Journal needs updates for v5.1.0

### PR DESCRIPTION
**Pull request by semalaiappan**
Added Test method for Journel Entry for Sql server and postgreSql
Added DatabaseReference for  5.1  PgSQl which was missing before 
Ed-Fi-MigrationUtility\EdFi.Ods.Utilities.Migration\DatabaseReference\5.1\EdFi.Ods.Standard\Artifacts\PgSql

### CODE REVIEW

**Comment by semalaiappan** [comment]
All test are pass in my local
testpass

File: EdFi.Ods.Utilities.Migration.Tests/MigrationTests/all_versions/EdFiOdsVersionJournalTests.cs [hunk 1]
```diff
@@ -20,7 +20,17 @@ public class EdFiOdsVersionJournalTests : DatabaseIntegrationTestBase
         private static readonly List<EdFiOdsVersion> VersionLevelConfigurationsUnderTest = GetVersionLevelConfigurationsUnderTest();
 
         [Test]
-        public void MigrationUtilityScriptsShouldBeUpToDateWithCurrentEdFiOdsBuild()
+        public void MigrationUtilityScriptsShouldBeUpToDateWithCurrentEdFiOdsBuildForSqlServer()
```
Comment (tmeador-dl) [inline EdFi.Ods.Utilities.Migration.Tests/MigrationTests/all_versions/EdFiOdsVersionJournalTests.cs]: I think we can revert the changes to this file, these should be covered by ODS-4750

File: EdFi.Ods.Utilities.Migration/VersionLevel/EdFiOdsVersionJournal.cs [hunk 2]
```diff
@@ -29,7 +29,9 @@ public class EdFiOdsVersionJournal
     {
         public string BaseVersionScriptPath { get; }
         public const string DefaultRelativeBaseScriptPath = "DatabaseReference";
-
```
Comment (tmeador-dl) [inline EdFi.Ods.Utilities.Migration/VersionLevel/EdFiOdsVersionJournal.cs]: I think we can revert the changes to this file, these should be covered by ODS-4750

**Review by tmeador-dl — APPROVED** [review]
